pkgs/by-name/hy/hydra/package.nix

```diff
@@ -48,6 +47,8 @@
 }:
 
 let
+  nix = nixVersions.nix_2_22;
```

Comment:
It would be nice if we could still override nix, sometimes I use an overlay to quickly verify some hydra patches and it could happen that nix also needed overriding then. Of course `nixVersions.nix_2_22 = nixVersions.nix_2_24` might work, but it doesn't feel great...

Not the highest priority, we can let it go if we don't find a satisfactory solution.

Reply:
Yeah, better to keep that around. Still using `all-packages` seems to be fine according to the docs.

https://github.com/NixOS/nixpkgs/blob/3c7d6a1dbcef3a281b48de2ad62a09dbba708b8a/pkgs/by-name/README.md?plain=1#L68-L69

https://github.com/NixOS/nixpkgs/blob/3c7d6a1dbcef3a281b48de2ad62a09dbba708b8a/pkgs/by-name/README.md?plain=1#L92